### DISCUSSION TITLE
how to access table cell by row & col index

### DISCUSSION THREAD

**Discussion by dkuo123** [ocornut/imgui#5353] [New Users Build/Link/Run issues ONLY!]
I created a table with tree_view, 
calculate the total. I can add a Total line after processing all the previous lines. But would like to go back and put the total amount in tree node.
so basically, I want to access the widget in previous row_idx, col_idx and write value there. is it possible?
![Screenshot 1](https://user-images.githubusercontent.com/5854314/170795367-99a2e831-fc23-4512-a7cb-746ae620e8bb.jpg)

**Comment by rokups** [comment]
It is not possible to append data to a previously rendered row. You should calculate total ahead of time and render it first.
